From 77d54a82da83a052a85cbb080ea6931c50e02843 Mon Sep 17 00:00:00 2001
From: Greg McCarthy <greg@gjmccarthy.co.uk>
Date: Thu, 11 Jul 2019 19:04:53 +0100
Subject: [PATCH] Revert "[format] fix include order"

This reverts commit d324e77adc7b31bc9ebd5b4e13513b7fcdcd4bd1.

diff --git a/xbmc/platform/linux/ScreenshotSurfaceRBP.cpp b/xbmc/platform/linux/ScreenshotSurfaceRBP.cpp
index 2c9e0a3b66..def422a19d 100644
--- a/xbmc/platform/linux/ScreenshotSurfaceRBP.cpp
+++ b/xbmc/platform/linux/ScreenshotSurfaceRBP.cpp
@@ -8,9 +8,8 @@
 
 #include "ScreenshotSurfaceRBP.h"
 
-#include "utils/Screenshot.h"
-
 #include "platform/linux/RBP.h"
+#include "utils/Screenshot.h"
 
 void CScreenshotSurfaceRBP::Register()
 {
diff --git a/xbmc/rendering/dx/ScreenshotSurfaceWindows.cpp b/xbmc/rendering/dx/ScreenshotSurfaceWindows.cpp
index cd97e57def..756bc6d098 100644
--- a/xbmc/rendering/dx/ScreenshotSurfaceWindows.cpp
+++ b/xbmc/rendering/dx/ScreenshotSurfaceWindows.cpp
@@ -8,13 +8,13 @@
 
 #include "ScreenshotSurfaceWindows.h"
 
-#include "ServiceBroker.h"
 #include "guilib/GUIComponent.h"
 #include "guilib/GUIWindowManager.h"
 #include "rendering/dx/DeviceResources.h"
+#include "ServiceBroker.h"
 #include "threads/SingleLock.h"
-#include "utils/Screenshot.h"
 #include "utils/log.h"
+#include "utils/Screenshot.h"
 #include "windowing/GraphicContext.h"
 
 #include <wrl/client.h>
diff --git a/xbmc/rendering/gl/ScreenshotSurfaceGL.cpp b/xbmc/rendering/gl/ScreenshotSurfaceGL.cpp
index 1c5589d1b4..be0bd892dd 100644
--- a/xbmc/rendering/gl/ScreenshotSurfaceGL.cpp
+++ b/xbmc/rendering/gl/ScreenshotSurfaceGL.cpp
@@ -8,17 +8,17 @@
 
 #include "ScreenshotSurfaceGL.h"
 
-#include "ServiceBroker.h"
 #include "guilib/GUIComponent.h"
 #include "guilib/GUIWindowManager.h"
+#include "ServiceBroker.h"
 #include "threads/SingleLock.h"
 #include "utils/Screenshot.h"
 #include "windowing/GraphicContext.h"
 
-#include <vector>
-
 #include "system_gl.h"
 
+#include <vector>
+
 void CScreenshotSurfaceGL::Register()
 {
   CScreenShot::Register(CScreenshotSurfaceGL::CreateSurface);
diff --git a/xbmc/rendering/gles/ScreenshotSurfaceGLES.cpp b/xbmc/rendering/gles/ScreenshotSurfaceGLES.cpp
index 2efc744ec7..7d7a9409c2 100644
--- a/xbmc/rendering/gles/ScreenshotSurfaceGLES.cpp
+++ b/xbmc/rendering/gles/ScreenshotSurfaceGLES.cpp
@@ -8,17 +8,17 @@
 
 #include "ScreenshotSurfaceGLES.h"
 
-#include "ServiceBroker.h"
 #include "guilib/GUIComponent.h"
 #include "guilib/GUIWindowManager.h"
+#include "ServiceBroker.h"
 #include "threads/SingleLock.h"
 #include "utils/Screenshot.h"
 #include "windowing/GraphicContext.h"
 
-#include <vector>
-
 #include "system_gl.h"
 
+#include <vector>
+
 void CScreenshotSurfaceGLES::Register()
 {
   CScreenShot::Register(CScreenshotSurfaceGLES::CreateSurface);
diff --git a/xbmc/utils/Screenshot.cpp b/xbmc/utils/Screenshot.cpp
index 638ea64832..c16e001ae4 100644
--- a/xbmc/utils/Screenshot.cpp
+++ b/xbmc/utils/Screenshot.cpp
@@ -8,19 +8,20 @@
 
 #include "Screenshot.h"
 
-#include "ServiceBroker.h"
-#include "URL.h"
-#include "Util.h"
 #include "filesystem/File.h"
 #include "guilib/LocalizeStrings.h"
 #include "pictures/Picture.h"
+#include "ServiceBroker.h"
 #include "settings/SettingPath.h"
 #include "settings/Settings.h"
 #include "settings/SettingsComponent.h"
 #include "settings/windows/GUIControlSettings.h"
+
+#include "URL.h"
+#include "Util.h"
 #include "utils/JobManager.h"
-#include "utils/URIUtils.h"
 #include "utils/log.h"
+#include "utils/URIUtils.h"
 
 using namespace XFILE;
 
-- 
2.16.4

