From 76337c3ed14268cdd955d001d1cfee7d2b7789e2 Mon Sep 17 00:00:00 2001
From: sarbes <sarbes@kodi.tv>
Date: Thu, 7 May 2020 20:29:46 +0200
Subject: [PATCH] fixed faulty thumbnail hashing

---
 xbmc/addons/interfaces/Filesystem.cpp | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/xbmc/addons/interfaces/Filesystem.cpp b/xbmc/addons/interfaces/Filesystem.cpp
index a4c72f0d6d98..0664c98b36bd 100644
--- a/xbmc/addons/interfaces/Filesystem.cpp
+++ b/xbmc/addons/interfaces/Filesystem.cpp
@@ -301,10 +301,9 @@ char* Interface_Filesystem::get_cache_thumb_name(void* kodiBase, const char* fil
     return nullptr;
   }
 
-  Crc32 crc;
-  crc.ComputeFromLowerCase(filename);
-  std::string string = StringUtils::Format("%08x.tbn", static_cast<unsigned int>(crc));
-  char* buffer = strdup(string.c_str());
+  const auto crc = Crc32::ComputeFromLowerCase(filename);
+  const auto hex = StringUtils::Format("%08x.tbn", crc);
+  char* buffer = strdup(hex.c_str());
   return buffer;
 }
 
