From d347245205c555fc0e277b6d4cc536e0d1a97bcf Mon Sep 17 00:00:00 2001
From: DaveTBlake <oak99sky@yahoo.co.uk>
Date: Fri, 3 Jul 2020 23:32:38 +0100
Subject: [PATCH] Fix check that another artist with this mbid not already
 exist

---
 xbmc/music/MusicDatabase.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/xbmc/music/MusicDatabase.cpp b/xbmc/music/MusicDatabase.cpp
index c6aa125d4b93..3639722ee8e2 100644
--- a/xbmc/music/MusicDatabase.cpp
+++ b/xbmc/music/MusicDatabase.cpp
@@ -1787,7 +1787,7 @@ int  CMusicDatabase::UpdateArtist(int idArtist,
   bool isScrapedMBID = bScrapedMBID;
   std::string artistname;
   int idArtistMbid = GetArtistFromMBID(strMusicBrainzArtistID, artistname);
-  if (idArtistMbid > 0)
+  if (idArtistMbid > 0 && idArtistMbid != idArtist)
   {
     CLog::Log(LOGDEBUG, "{0}: Updating {4} (Id: {5}) mbid {1} already assigned to {2} (Id: {3})",
       __FUNCTION__, strMusicBrainzArtistID.c_str(), artistname.c_str(), idArtistMbid,
@@ -1836,10 +1836,10 @@ bool CMusicDatabase::UpdateArtistScrapedMBID(int idArtist, const std::string& st
   if (strMusicBrainzArtistID.empty() || idArtist < 0)
     return false;
 
-  // Check artist with this mbid not already exist (an alias for example)
+  // Check another artist with this mbid not already exist (an alias for example)
   std::string artistname;
   int idArtistMbid = GetArtistFromMBID(strMusicBrainzArtistID, artistname);
-  if (idArtistMbid > 0)
+  if (idArtistMbid > 0 && idArtistMbid != idArtist)
   {
     CLog::Log(LOGDEBUG, "{0}: Artist mbid {1} already assigned to {2} (Id: {3})", __FUNCTION__,
               strMusicBrainzArtistID.c_str(), artistname.c_str(), idArtistMbid);
