From edc023184cb3602bd86c4f625567b820bf8560f0 Mon Sep 17 00:00:00 2001
From: Andrii Liubimov <andrii.liubimov@gmail.com>
Date: Mon, 6 Jul 2020 11:29:46 -0400
Subject: [PATCH] Fix matching video for GLES

---
 xbmc/windowing/X11/WinSystemX11GLESContext.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xbmc/windowing/X11/WinSystemX11GLESContext.cpp b/xbmc/windowing/X11/WinSystemX11GLESContext.cpp
index 016918df8cf1..ed72f9444d3d 100644
--- a/xbmc/windowing/X11/WinSystemX11GLESContext.cpp
+++ b/xbmc/windowing/X11/WinSystemX11GLESContext.cpp
@@ -259,6 +259,8 @@ XVisualInfo* CWinSystemX11GLESContext::GetVisual()
   }
 
   XVisualInfo x11_visual_info_template;
+  memset(&x11_visual_info_template, 0, sizeof(XVisualInfo));
+
   if (!eglGetConfigAttrib(eglDisplay, eglConfig,
     EGL_NATIVE_VISUAL_ID, reinterpret_cast<EGLint*>(&x11_visual_info_template.visualid)))
   {
