From 29ed357563308ac46ee919d56af3f9b62372c9ed Mon Sep 17 00:00:00 2001
From: ronie <ronie@kodi.tv>
Date: Tue, 30 Jun 2020 17:40:49 +0200
Subject: [PATCH] [filemanger] reset to root if directory is inaccessible

---
 xbmc/windows/GUIWindowFileManager.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xbmc/windows/GUIWindowFileManager.cpp b/xbmc/windows/GUIWindowFileManager.cpp
index e5e0971f538b..f5ca1abe8d94 100644
--- a/xbmc/windows/GUIWindowFileManager.cpp
+++ b/xbmc/windows/GUIWindowFileManager.cpp
@@ -1245,11 +1245,13 @@ void CGUIWindowFileManager::OnInitWindow()
   else if (!bResult0)
   {
     ShowShareErrorMessage(m_Directory[0]); //show the error message after window is loaded!
+    Update(0, ""); // reset view to root
   }
 
   if (!bResult1)
   {
     ShowShareErrorMessage(m_Directory[1]); //show the error message after window is loaded!
+    Update(1, ""); // reset view to root
   }
 }
 
